blocks/article-hero-video/article-hero-video.css

```diff
@@ -0,0 +1,124 @@
+div.article-hero-video-container {
+    overflow: hidden;
+}
+
+div.article-hero-video {
+    width: 100vw;
+    height: auto;
+    aspect-ratio: 16/9;
+    border: none;
+    margin: 0;
+    animation-timing-function: linear;
+    display: block;
+    transform: scaleX(1.1) scaleY(1.1)
```

Comment:
The original hides the youtube logo with a combination of heigh, overflow and negative margins. You can try the same here 
![Screenshot 2023-08-24 at 11 32 35](https://github.com/hlxsites/24life/assets/2250964/f64fa1e0-06d9-4ac6-8d08-48a6c7710e17)